src/main/java/org/opentripplanner/routing/edgetype/SimpleTransfer.java

```diff
@@ -48,6 +48,10 @@ public State traverse(State s0) {
         if(distance > s0.getOptions().maxTransferWalkDistance) {
             return null;
         }
+        // Don't allow SimpleTransfer right after Call-n-Ride - in that case we need to transfer at the same stop
```

Comment:
This restriction applies to transfers after both call-and-ride and deviated-route dropoffs. We specifically disallow a SimpleTransfer from a point at which the vehicle makes a special stop for the passenger. It wouldn't really make sense to get dropped off at stop A (where the vehicle doesn't normally stop there, but did via a flexible segment), and then walk 10 meters to stop B to take a fixed-route service -- you would simply get dropped off at stop B in the first place. But since call-and-ride service has a large reluctance associated with it, we were getting itineraries that looked like that. We considered this a perverse result.